Core/Src/main.c

```diff
@@ -526,6 +536,217 @@ static void MX_SPI3_Init(void)
 
 }
 
+/**
+  * @brief TIM1 Initialization Function
+  * @param None
+  * @retval None
+  */
+static void MX_TIM1_Init(void)

```

Comment:
@nwdepatie might have different thought, but my experience in pulling the low level MX functions and defining them at app level has been sketchy at best. In this case, im guessing this will be called automatically during the timer inintialzation in the main function, and redefining and calling it here might not be needed